src/components/Landing/FAQSection/FAQSection.module.css

```diff
@@ -0,0 +1,57 @@
+.wrapper {
```

Comment:
try to shorten this file and keep really only the styles which are not possible to do in mantine